File: drs-backend/drs-google-sheet/src/main/java/com/kindminds/drs/BizCoreCtx.java
package com.kindminds.drs;

import com.kindminds.drs.service.util.DrsBizCoreAnnotationConfig;
import org.springframework.context.ApplicationContext;
import org.springframework.context.annotation.AnnotationConfigApplicationContext;

public class BizCoreCtx {

    //todo here
    private static  AnnotationConfigApplicationContext springCtx =
            new AnnotationConfigApplicationContext(DrsBizCoreAnnotationConfig.class);


    public static AnnotationConfigApplicationContext get() {
        return springCtx;
    }
}

File: drs-data-pipelines/core/src/main/scala/com/kindminds/drs/data/pipelines/core/BizCoreCtx.scala
package com.kindminds.drs.data.pipelines.core


import com.kindminds.drs.service.util.DrsBizCoreAnnotationConfig
import org.springframework.context.annotation.AnnotationConfigApplicationContext

object BizCoreCtx {

  //todo arthur
  val springCtx: AnnotationConfigApplicationContext =
    new AnnotationConfigApplicationContext(classOf[DrsBizCoreAnnotationConfig])

  def get(): AnnotationConfigApplicationContext = {

    springCtx

  }

}
